rosette/src/CommandLine.cc

```diff
@@ -58,7 +59,7 @@ int RestoringImage = 0;
 void usage(const char* name) {
     suicide(
         "Usage: %s [-t num] [-p [num]] [-is num] [-ss num] [-os num] [-boot "
-        "filename]",
+        "filename] [-run filename]",
```

Comment:
Usually, the file to be executed is a positional argument and not a flag. Is there any way to do that?

Reply:
Just making this positional can cause confusion as to exactly which command line parameters are passed on to the RBL. We are investigating other options (perhaps the google command line parser that is used in pyr8).